### ISSUE TITLE
ADA Compliant Links on Social Component

### ISSUE THREAD

**Issue by minimart** [reactioncommerce/reaction#2182]
### Proposed Enhancement
Social links do not currently comply with ADA or [ WCAG 2.0](https://www.w3.org/TR/WCAG20/) requirements. These links (Facebook, Pinterest, Google Plus, and Twitter) should be ADA/WCAG 2.0 compliant by having text readable by screen readers.

### Benefit to Reaction and Users 
Compliance with these standards is legally mandated in the United States, EU and several other governing bodies. 

### Proposed changes
In imports/plugins/included/social/client/components adding an aria-label to each component, ex:

`<a className="btn btn-flat facebook-share" href="#" onClick={this.handleClick} aria-label="Share to Facebook" target="_blank">
        <Helmet
          meta={getOpenGraphMeta(this.props)}
        />
        <i className={iconClassNames} />
        {this.renderText()}
      </a>`

I will begin to work on this issue, and create a PR for these simple changes to the social plugin when completed.

**Comment by aaronjudd** [comment]
👍 Excellent points.  Related to #405, which we're trying to give a higher priority.

**Comment by kieckhafer** [comment]
Closed in PR #2196
